packages/core/src/llm/openai.ts

```diff
@@ -71,6 +31,16 @@ function createOpenAIClient(props: CreateOpenAIClientProps) {
   return new OpenAI(openAiFields);
 }
 
+function createOpenAIProvider(props: CreateOpenAIClientProps): OpenAIProvider {
```

Comment:
"ai" now give us an OpenAIProvider, so we export both this and the original OpenAI